container/src/LuigiCompoundContainer.svelte

```diff
@@ -104,6 +106,7 @@
       nodeParams &&
       pathParams &&
       searchParams &&
+      skipInitCheck &&
```

Comment:
I think there needs to be some logic here to use the **skipInitCheck** set variable to "skip the init check" when **CompoundContainer** is first built. Similar to how **skipInitCheck** is used in **LuigiContainer**.